parsers/gdscript.c

```diff
@@ -0,0 +1,784 @@
+/*
+*   Copyright (c) 2000-2003, Darren Hiebert
```

Comment:
You are not Darren Hiebert.

Reply:
This year is 2021.

Reply:
I was somewhat conflicted about whether to change it because this used so much of his code from the python one. I figured that I'd leave it since if I had made the same amount of changes in a patch to the python file I'd surely have left that line alone. Do I just replace it anyway, or do I add myself after it as an addendum?

Reply:
Oh, I see. 
I'm not sure how it should be. 
How about this one?

```
/*
*   Copyright (c) 2000-2003, Darren Hiebert
*   Copyright (c) 2014-2016, Colomban Wendling <ban@herbesfolles.org>
*   Copyright (c) 2021, David Yang
*
*   This source code is released for free distribution under the terms of the
*   GNU General Public License version 2 or (at your option) any later version.
*
*   This module contains functions for generating tags for GDScript language
*   files. This module is derived from the Python module.
*
* 	GDScript language reference:
* 	https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_basics.html
*
*/
```

Reply:
Updated because the GDScript parser is based on the line-oriented older Python parser.

Reply:
Yeah sure, I had modified before my name but I can switch to fancy copyright. The first line'll probably change I think, if I'm switching to using the new python parser as base